atc/gc/build_log_collector.go

```diff
@@ -123,6 +130,12 @@ func (br *buildLogCollector) Run(ctx context.Context) error {
 					}
 				}
 
+				if daysToRetainBuildLogs > 0 {
+					if build.StartTime().AddDate(0, 0, daysToRetainBuildLogs).After(time.Now()) {
```

Comment:
Should this be `build.FinishTime()`? It might matter for builds that take a very long time to run - if it runs for 2 days but I have a policy of 'reap after 1 day' I wouldn't want it to be immediately reaped.